azure-pipelines/signing.yml

```diff
@@ -341,6 +358,10 @@ jobs:
           move "$(Build.ArtifactStagingDirectory)\staging\vcpkg-muslc" "$(Build.ArtifactStagingDirectory)\drop\vcpkg-muslc"
           move "$(Build.ArtifactStagingDirectory)\staging\vcpkg-standalone-bundle.tar.gz" "$(Build.ArtifactStagingDirectory)\drop\vcpkg-standalone-bundle.tar.gz"
 
+          move /Y "$(Build.ArtifactStagingDirectory)\staging\applocal.ps1" "$(Build.ArtifactStagingDirectory)\vs-insertion\staging\scripts\buildsystems\msbuild\applocal.ps1"

```

Comment:
I think you need to edit mint-standalone-bundle.ps1 to pick up these signed copies so they also end up in vcpkg-standalone-bundle.tar.gz. Other than that this looks great to me!